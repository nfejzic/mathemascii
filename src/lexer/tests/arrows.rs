use crate::lexer::tests::Snapshot;

super::test_snap!(right, "rarr rightarrow");
super::test_snap!(big_left, "lArrLeftarrow");
super::test_snap!(too_long, "abcdefghijklmnopqrstuvwxy");
super::test_snap!(multiple, "uarr->|->twoheadrightarrowtailto");
