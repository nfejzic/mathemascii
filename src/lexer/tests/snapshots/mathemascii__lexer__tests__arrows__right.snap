---
source: src/lexer/tests/arrows.rs
expression: "Snapshot((input, tokens))"
---
rarr rightarrow

rarr
^^^^ -> Arrow(Right) at: 0 -> 4

     rightarrow
     ^^^^^^^^^^ -> Arrow(Right) at: 5 -> 15


